### PR TITLE
Patch 1

### PR DESCRIPTION
**Pull request by lexuantrung-Peter**

### CODE REVIEW

File: Bin/01_most_words_found.rb [hunk 1]
```diff
@@ -0,0 +1,14 @@
+# @param {String[]} sentences
+# @return {Integer}
+def most_words_found(sentences)
+    max = 0
+    sentences.each do |s|
+        # puts("ssss")
+        length = s.split(' ').length 
+        if max < length
+            max = length
```
Comment (lexuantrung-Peter) [inline Bin/01_most_words_found.rb]:
```suggestion
            max = length if max < length
```

File: Bin/blocks_06.rb [hunk 2]
```diff
@@ -0,0 +1,23 @@
...
+  "The Legend of Zelda: Majora's Mask" => 0.9195, 
+  "1080: TenEighty Snowboarding" => 0.8960, 
+  "Conker's Bad Fur Day" => 0.8928, 
+  "Excitebike 64" => 0.8907, 
+  "Turok 2: Seeds of Evil" => 0.8896, 
+  "Paper Mario" => 0.8881
+}
+
+# Your code goes here
+total_scores = game_rankings.map { |k,v| v}.reduce(0, :+)
```
Comment (lexuantrung-Peter) [inline Bin/blocks_06.rb:17]:
```suggestion
total_scores = game_rankings.values.sum
```

File: Bin/book.rb [hunk 3]
```diff
@@ -0,0 +1,56 @@
...
+    books = []
+
+    loop do
+
+      data = search_keyword(book_name, page)
+      data['books'].each do |book|
+        isbn = book['isbn13']
+        b = get_book_detail(isbn)
+        puts b.title
+        books << b
```
Comment (lexuantrung-Peter) [inline Bin/book.rb]:
```suggestion
        books << get_book_detail(book['isbn13']
```

File: Bin/01_most_words_found.rb (hunk 1 above)
Comment (huuvinhnguyen) [reply Bin/01_most_words_found.rb]: fixed, thank you!

File: Bin/blocks_06.rb (hunk 2 above)
Comment (huuvinhnguyen) [reply Bin/blocks_06.rb:17]: updated, thank you!

File: Bin/book.rb (hunk 3 above)
Comment (huuvinhnguyen) [reply Bin/book.rb]: updated, thank you!

**Pull request merged**